packages/server/lib/util/ci_provider.js

```diff
@@ -38,6 +38,12 @@ const isCodeshipPro = () => {
   return process.env.CI_NAME && (process.env.CI_NAME === 'codeship') && !process.env.CODESHIP
 }
 
+const isConcourse = () => {
+  return (_.filter(Object.keys(process.env), (key) => {
```

Comment:
Should use `_.some()` here, which stops iteration when the predicate fn returns true.